diesel_cli/src/print_schema.rs

```diff
@@ -118,6 +134,20 @@ impl<'a> Display for ColumnDefinitions<'a> {
     }
 }
 
+struct Joinable<'a>(&'a ForeignKeyConstraint);
+
+impl<'a> Display for Joinable<'a> {
+    fn fmt(&self, f: &mut Formatter) -> fmt::Result {
+        writeln!(
+            f,
+            "joinable!({} -> {} ({}));",
```

Comment:
I'm going to run it on the whole codebase one of these days. Trying to reduce the churn in the short term